recipes/repeatfinder/build.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-$CC -std=c99 -g -o repeatfinder main.c conversion.* switches.*
+gcc -std=c99 -g -o repeatfinder main.c conversion.* switches.*
```

Comment:
using `$CC` is correct here, different platforms have different compilers as explained here: https://bioconda.github.io/contributor/troubleshooting.html#g-or-gcc-not-found